packages/esm-patient-queues-app/src/active-visits/change-status-dialog.component.tsx

```diff
@@ -66,20 +63,35 @@ const ChangeStatus: React.FC<ChangeStatusDialogProps> = ({ queueEntry, currentEn
 
   const { activeVisit } = useVisit(queueEntry.patientUuid);
 
-  getCareProvider(sessionUser?.user?.uuid).then(
-    (response) => {
-      setProvider(response?.data?.results[0].uuid);
-      mutate();
-    },
-    (error) => {
-      showNotification({
-        title: t(`errorGettingProvider', 'Couldn't get provider`),
-        kind: 'error',
-        critical: true,
-        description: error?.message,
-      });
-    },
-  );
+  const [isLoading, setIsLoading] = useState(true);
+
+  const providerUuid = useMemo(() => {
+    if (!sessionUser?.user?.uuid) return null;
+    setIsLoading(true);
```

Comment:
By default, you have set the state of isLoading to true and you're setting it here again as true, was this intentional or it should be the other way round?

Reply:
intentional

Reply:
Okay then. Do you mind pulling the latest changes from master am getting 179 conflicts in the yarn.lock file

Reply:
you definitely have a separate lock file.

Reply:
Because my lock file is up to date.